source/dotnet/Library/AdaptiveCards/AdaptiveCards.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageProjectUrl>https://adaptivecards.io</PackageProjectUrl>
     <PackageIconUrl>https://adaptivecards.io/content/icons_blue/blue-48.png</PackageIconUrl>
     <RepositoryUrl>https://github.com/Microsoft/AdaptiveCards</RepositoryUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/AdaptiveCards/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
><PackageLicenseExpression>MIT</PackageLicenseExpression> [](start = 4, length = 56)

why is this set to the MIT value?

Reply:
The nuspec goo emitted by `PackageLicenseUrl` is [being deprecated](https://docs.microsoft.com/en-us/nuget/reference/nuspec), and tooling was complaining about it. Instead, if you use a [canonical well-known license](https://spdx.org/licenses/), you use the license type. Otherwise if you have a custom license, you have to embed the license as a file in the project and refer to it directly.

---
In reply to: [282977498](https://github.com/microsoft/AdaptiveCards/pull/2832#discussion_r282977498) [](ancestors = 282977498)